modules/exploits/unix/webapp/wp_infinitewp_auth_bypass.rb

```diff
@@ -0,0 +1,121 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = ExcellentRanking
```

Comment:
`ExcellentRanking` for a module which clobbers files?

Reply:
I'll drop it if I don't get to the to-do listed in the description. This is marked `delayed` precisely for that reason.

Reply:
IIRC, the content can simply be read from a `<textarea name="newcontent">` tag, then we can prepend our payload and execute.

Reply:
On second thought, I think I might drop it to `NormalRanking` now and leave it there. Editing an existing file has a little room for error.